idk/serverless/importer.go

```diff
@@ -33,19 +33,28 @@ func NewImporter(controller dax.Controller, qdbid dax.QualifiedDatabaseID, tbl *
 	}
 }
 
-// fbClient currently returns a new FeatureBase client (for the address) for
-// every call to this method. We could cache these connections in a map (keyed
-// on address) to avoid creating a new client for an address that we already
-// have a client for.
+var fbClientCache = map[dax.Address]*featurebaseclient.Client{}
+var fbClientCacheMu sync.Mutex
+
 func (m *importer) fbClient(address dax.Address) (*featurebaseclient.Client, error) {
-	// Set up a FeatureBase client with address.
-	return featurebaseclient.NewClient(address.HostPort(),
+	fbClientCacheMu.Lock()
+	defer fbClientCacheMu.Unlock()
+	client := fbClientCache[address]
+	if client != nil {
+		return client, nil
+	}
+	client, err := featurebaseclient.NewClient(address.HostPort(),
 		featurebaseclient.OptClientRetries(2),
 		featurebaseclient.OptClientTotalPoolSize(1000),
 		featurebaseclient.OptClientPoolSizePerRoute(400),
 		featurebaseclient.OptClientPathPrefix(address.Path()),
-		//featurebaseclient.OptClientStatsClient(m.stats),
+		//featurebase_client.OptClientStatsClient(m.stats),
```

Comment:
uh, why underscore?

Reply:
So, if you look at the other copy of `fbClient`, it has underscores, and i cut and pasted it here and then edited it, so i didn't catch this.